package.json

```diff
@@ -1,5 +1,32 @@
 {
+  "name": "istanbuljs",
+  "version": "1.0.0",
```

Comment:
@JaKXz slightly confusing I know, but we've been using `2.x` to refer to the process of splitting Istanbul into a series of packages rather than one monolith -- I don't disagree with you, but I think it might not be worth a major bump since several libraries rely on these modules already and it would be confusing to them.